src/workerd/jsg/fast-method-test.c++

```diff
@@ -0,0 +1,304 @@
+#include "jsg-test.h"
+
+namespace workerd::jsg::test {
+namespace {
+
+class FastMethodContext: public jsg::Object, public jsg::ContextGlobal {
+ public:
+  int32_t add(int32_t a, int32_t b) {
+    return a + b;
+  }
+
+  int32_t processValue(v8::Local<v8::Value> value) {
+    // Return 42 if it's a number, 0 otherwise
+    if (value->IsNumber()) {
+      return 42;
```

Comment:
Why not actually return the number? Wouldn't it be useful to verify we're getting the _correct_ value?

Reply:
Agreed. That sounds better.